pkg/sentry/pgalloc/pgalloc.go

```diff
@@ -1095,6 +1097,16 @@ func (f *MemoryFile) findReclaimable() (platform.FileRange, bool) {
 		for seg := f.usage.LowerBoundSegment(f.minReclaimablePage); seg.Ok(); seg = seg.NextSegment() {
 			if seg.ValuePtr().refs == 0 {
 				f.minReclaimablePage = seg.End()
+				if f.reclaimableSize > seg.Range().Length() {
```

Comment:
Shouldn't f.reclaimableSize always be >= seg.Range().Length()? Why not just unconditionally do `f.reclaimableSize -= seg.Range.Length()`?